tests/unit/features/partitions/test_parts.py

```diff
@@ -157,59 +154,53 @@ def test_part_invalid_partition_usage_complex(
+            - unknown partition '123' in '(123)/test/(default)' in field 'parts.part-a.stage'
+            - unknown partition '123' in '(123)/test/(kernel)' in field 'parts.part-a.stage'
+            - unknown partition '' in '()' in field 'parts.part-a.prime'
+            - unknown partition 'foo' in '(foo)' in field 'parts.part-a.prime'
+            - unknown partition 'bar' in '(bar)/test' in field 'parts.part-a.prime'
+            - unknown partition 'baz' in '(baz)' in field 'parts.part-a.prime'
+            - unknown partition '123' in '(123)/test' in field 'parts.part-a.prime'
+            - unknown partition '123' in '(123)/test/(456)' in field 'parts.part-a.prime'
+            - unknown partition '123' in '(123)/test/(default)' in field 'parts.part-a.prime'
+            - unknown partition '123' in '(123)/test/(kernel)' in field 'parts.part-a.prime'
+            - unknown partition '' in '()' in field 'parts.part-b.organize'
+            - unknown partition 'foo' in '(foo)' in field 'parts.part-b.organize'
+            - unknown partition 'bar' in '(bar)/test' in field 'parts.part-b.organize'
+            - unknown partition 'baz' in '(baz)' in field 'parts.part-b.organize'
+            - unknown partition '123' in '(123)/test' in field 'parts.part-b.organize'
+            - unknown partition '123' in '(123)/test/(456)' in field 'parts.part-b.organize'
+            - unknown partition '123' in '(123)/test/(default)' in field 'parts.part-b.organize'
+            - unknown partition '123' in '(123)/test/(kernel)' in field 'parts.part-b.organize'
+            - unknown partition '' in '()' in field 'parts.part-b.stage'
+            - unknown partition 'foo' in '(foo)' in field 'parts.part-b.stage'
+            - unknown partition 'bar' in '(bar)/test' in field 'parts.part-b.stage'
+            - unknown partition 'baz' in '(baz)' in field 'parts.part-b.stage'
+            - unknown partition '123' in '(123)/test' in field 'parts.part-b.stage'
+            - unknown partition '123' in '(123)/test/(456)' in field 'parts.part-b.stage'
+            - unknown partition '123' in '(123)/test/(default)' in field 'parts.part-b.stage'
+            - unknown partition '123' in '(123)/test/(kernel)' in field 'parts.part-b.stage'
+            - unknown partition '' in '()' in field 'parts.part-b.prime'
+            - unknown partition 'foo' in '(foo)' in field 'parts.part-b.prime'
+            - unknown partition 'bar' in '(bar)/test' in field 'parts.part-b.prime'
+            - unknown partition 'baz' in '(baz)' in field 'parts.part-b.prime'
+            - unknown partition '123' in '(123)/test' in field 'parts.part-b.prime'
+            - unknown partition '123' in '(123)/test/(456)' in field 'parts.part-b.prime'
+            - unknown partition '123' in '(123)/test/(default)' in field 'parts.part-b.prime'
+            - unknown partition '123' in '(123)/test/(kernel)' in field 'parts.part-b.prime'
```

Comment:
Maybe we could keep the structured view of the previous approach. It seems easier to read, wdyt?

Reply:
Yes, I liked it too.  I went back to the previous structured view but with the dot (.) notation you suggested in an earlier comment.